Add a list of hyphenated Brazilian Portuguese words

filipenegrao opened the pull request:
A list of more than 25,000 words in Brazilian Portuguese, all checked using python and a popular online dictionary.

felipesanches commented:
valeu, Filipe! Vou dar uma olhada amanhã nisso ;-)

felipesanches commented:
Thanks, Petr!

petrvanblokland commented:
Filipe, 

Thanks for adding the list.

Felipe, Filipe,

I added some docTest for Portugese. 
I don’t know it if makes sense to look for combined words, as in Dutch, and I added a random one to show that it works.
If words really get combined, can you add some real ones?

It’s in hyphenation.py

P

> On 21 Mar 2018, at 04:24, Felipe Corrêa da Silva Sanches <notifications@github.com> wrote:
> 
> Thanks, Petr!
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/TypeNetwork/PageBot/pull/122#issuecomment-374824572>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAXBwdGkATSkMqSvM6wXlTLpTdAuO3Xdks5tgcgAgaJpZM4SyuOB>.
> 

-----------------------------------------------
Petr van Blokland | twitter+skype @petrvanblokland
Intentionally not on FaceBook.
buro@petr.com | www.petr.com
mobile +31 6 24 219 502 | buro +31 15 887 1233 
-----------------------------------------------
Rietveld 56 2611 LM Delft The Netherlands
-----------------------------------------------
Claudia Mens | claudia@petr.com 
mobile +31 6 41 367 689 | buro +31 15 887 1233
-----------------------------------------------

filipenegrao commented:
Petr, 

In Portuguese, we don’t have that kind of combined words like in Dutch (and German as well?). It is possible to combine some words, but they are very specific like ‘econômico-social' (social economic), and they usually become another word, i.e., they could be treated inside a dictionary as a new/different word.

Filipe

> On 21 Mar 2018, at 04:45, Petr van Blokland <notifications@github.com> wrote:
> 
> Filipe, 
> 
> Thanks for adding the list.
> 
> Felipe, Filipe,
> 
> I added some docTest for Portugese. 
> I don’t know it if makes sense to look for combined words, as in Dutch, and I added a random one to show that it works.
> If words really get combined, can you add some real ones?
> 
> It’s in hyphenation.py
> 
> P
> 
> > On 21 Mar 2018, at 04:24, Felipe Corrêa da Silva Sanches <notifications@github.com> wrote:
> > 
> > Thanks, Petr!
> > 
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub <https://github.com/TypeNetwork/PageBot/pull/122#issuecomment-374824572>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAXBwdGkATSkMqSvM6wXlTLpTdAuO3Xdks5tgcgAgaJpZM4SyuOB>.
> > 
> 
> -----------------------------------------------
> Petr van Blokland | twitter+skype @petrvanblokland
> Intentionally not on FaceBook.
> buro@petr.com | www.petr.com
> mobile +31 6 24 219 502 | buro +31 15 887 1233 
> -----------------------------------------------
> Rietveld 56 2611 LM Delft The Netherlands
> -----------------------------------------------
> Claudia Mens | claudia@petr.com 
> mobile +31 6 41 367 689 | buro +31 15 887 1233
> -----------------------------------------------
> 
> 
> 
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/TypeNetwork/PageBot/pull/122#issuecomment-374855190>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABJJVln_3q4TCutI72BKga6rHdh9BfrQks5tggUAgaJpZM4SyuOB>.
>
